homework35/modules/validation.js

```diff
@@ -0,0 +1,79 @@
+const validationFunctions = {
+    isRequired: value => value === "" ? false : true,
+    isBetween: (length, min, max) => length < min || length > max ? false : true,
+
+    isDateOfBirth: (date) => {
+        const re = new RegExp("^((0[1-9])|(1[0-9])|(2[0-9])|(3[0-1]))/(0[1-9])|(1[0-2])$");
+        return re.test(date);
+    },
+
+    showError: (input, message) => {
```

Comment:
Had better splitting function into different files with different tasks

1. Validator functions,  the task is to validate a different data and return only `true` or `false` - let it be `validators.js`
2. Manage UI functions (like `showError`), the task is to renderer some parts of UI - let it be `form-helpers.js`, `ui-helpers.js` etc.